libr/core/core.c

```diff
@@ -1286,10 +1285,6 @@ static bool find_autocomplete(RLine *line) {
 		// handled before
 		break;
 	default:
-		if (p && !*p) {
```

Comment:
can't it be that `*p` is 0?

Reply:
```
			RCmdDescriptor *desc = &core->root_cmd_descriptor;
```
it has empty `help_msg`, no need for the special case.